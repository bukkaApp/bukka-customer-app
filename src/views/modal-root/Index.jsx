import React from 'react';

import PropTypes from 'prop-types';

import AuthModal from '../../components/navbar/common/AuthModal';
import InviteFriends from '../invite/InviteFriends';

export const Index = ({ push }) => (
  <>
    <AuthModal push={push} />
    <InviteFriends push={push} />
  </>
);

export default Index;

Index.propTypes = {
  push: PropTypes.func.isRequired,
};
