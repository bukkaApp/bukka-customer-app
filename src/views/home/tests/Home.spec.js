import React from 'react';
import Index from '..';

describe('Home Component', () => {
  const props = {
    history: {
      push: jest.fn(),
    }
  };

  const wrapper = shallow(<Index {...props} />);

  it('renders properly', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
