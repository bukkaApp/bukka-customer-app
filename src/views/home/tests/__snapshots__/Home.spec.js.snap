// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home Component renders properly 1`] = `ShallowWrapper {}`;
